client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -195,11 +211,6 @@ def get_generic_job_info(self, instance: pyblish.api.Instance):
         if job_info.SecondaryPool != "none":
             job_info.SecondaryPool = job_info.SecondaryPool
 
-        exp = instance.data.get("expectedFiles")
```

Comment:
This was removed, is it intentional?

Reply:
Yes - it's now called outside of `get_generic_job_info` here: https://github.com/ynput/ayon-deadline/pull/108/files#diff-eab3e29bdc024482042df8217ac8fb6fb97371171f83ecf95186aed7f39d4d28R111